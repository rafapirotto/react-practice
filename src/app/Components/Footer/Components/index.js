import FooterTitle from './FooterTitle';
import FooterItemList from './FooterItemList';

export { FooterTitle, FooterItemList };
