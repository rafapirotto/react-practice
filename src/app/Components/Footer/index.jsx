import React from 'react';

import { FooterItemList, FooterTitle } from './Components';

const Footer = () => (
  <footer className="linear-gradient">
    <FooterTitle title="Questions? Call +1 (408) 600-1722 (USA)" />
    <FooterItemList />
  </footer>
);

export default Footer;
