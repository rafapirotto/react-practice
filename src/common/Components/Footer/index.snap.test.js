import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import Footer from '.';

it('matches the snapshot', () => {
  const component = renderer.create(
    <Router>
      <Footer />
    </Router>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
