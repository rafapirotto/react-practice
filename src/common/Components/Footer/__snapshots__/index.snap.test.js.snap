// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `
<footer
  className="linear-gradient"
>
  <div
    className=" flex pt-24 text-4xl sm:justify-center sm:pl-0 md:justify-center md:pl-0 lg:justify-start lg:pl-40 "
  >
    Questions? Call +1 (408) 600-1722 (USA)
  </div>
  <div
    className=" mt-12 grid pb-40 sm:grid-cols-1 sm:pl-0 sm:text-center md:grid-cols-2 md:pl-0 md:text-center lg:grid-cols-4 lg:pl-40 lg:text-left "
  >
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        FAQ
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Help Center
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Account
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Media Center
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Privacy
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Jobs
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Ways to Watch
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Terms of Use
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Speed Test
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Cookie Preferences
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Corporate Information
      </a>
    </div>
    <div
      className="pb-2"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Contact Us
      </a>
    </div>
  </div>
</footer>
`;
